lib/information.ex

```diff
@@ -20,6 +40,7 @@ defmodule Xcribe.Information do
     end
   end
 
+  @doc false
```

Comment:
If we intend this macro to be used in by the users. It should be documented.

Reply:
Sure! Doc added :+1: